app/jobs/podcast_episodes/update_media_url_job.rb

```diff
@@ -0,0 +1,12 @@
+module PodcastEpisodes
+  class UpdateMediaUrlJob < ApplicationJob
+    queue_as :podcast_episode_update
+
+    # @param episode_id [Integer] - episode id
+    # @param enclosure_url [String] enclosure url from podcast RSS
+    def perform(episode_id, enclosure_url, update = Podcasts::UpdateEpisodeMediaUrl)
```

Comment:
I'd call the last parameter `servuce` just because all other jobs have that name